### PR TITLE
DEPRECATED: Grammar review Conclusion and  Evaluation

### PR DESCRIPTION
**Pull request by pavolloffay**
## Do not merge!

Current pdf: https://github.com/hawkular/hawkular-datamining/raw/doc/doc/thesis.pdf

### CODE REVIEW

File: doc/content2.tex [hunk 1]
```diff
@@ -1,723 +0,0 @@
...
-
-The module was successfully integrated into Hawkular and predictive charts of the metrics are located in Metric explorer
-tab.
-
-Future work on this module should analyze data from production servers and if necessary change the properties of
-optimizers in order to get more accurate predictions. Analysis could also contain evaluation if damped trend should be
-added to the implemented models.
-
-As it is open source project, it would be great to attract more people into development and continue with adding new
-models and features for time series analysis. Because in Java there is no widely used and maintained library.
```
Comment (grammar-review) [inline doc/content2.tex]: test

File: doc/content2.tex [hunk 2]
```diff
@@ -1,723 +0,0 @@
...
-        \end{figure}
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\chapter{Conclusion} \label{chap:conclusion}
-The goal of the master's thesis is to develop independent module in Java language for providing alert prediction
-capabilities for open source project Hawkular.
-
-%The thesis first describes various approaches for time series forecasting, and evaluates which methods are suitable
-%for Hawkular requirements. Follows review of existing monitoring solutions and libraries for time series modelling.
-%It is shown that any of existing libraries are not suitable for Hawkular, therefore a decision was made to implement
```
Comment (grammar-review) [inline doc/content2.tex:693]: therefore, - ciarka aj pred aj z slovom.

Comment (pavolloffay) [inline doc/content2.tex:693]: Janka ked je ten text taky sivy znamena to ze je to komentar (nie je to vo vyslednej praci)

Komentare v latexu sa robia znakom %

Comment (pavolloffay) [inline doc/content2.tex]: Komentare v latexu sa robia znakom %

Comment (grammar-review) [inline doc/content2.tex:693]: dakujem :)

File: doc/content2.tex [hunk 3]
```diff
@@ -1,723 +0,0 @@
...
-%selected models.
-
-Implementation part is split into multiple artifacts with dedicated functionality. The most important module
-focuses on time series modelling. It contains simple, double and triple exponential smoothing models which
-use non-linear optimization algorithm for finding the best parameters with goal of minimizing prediction error.
-The library also contains several time series utility classes and statistical tests for time series analysis.
-
-Data Mining prediction engine autonomously selects the best time series model for underlying time series. Most
-importantly, it is capable of dealing with concept drift problem. So in production there is no need of any
-analytical configuration.
```
Comment (grammar-review) [inline doc/content2.tex:703]: So there is no need of any analytical configuration in production.

File: doc/content2.tex (hunk 1 above)
Comment (grammar-review) [inline doc/content2.tex:723]: ...analysis, because there is no widely used and maintained library in Java. /slovosled

File: doc/thesis.tex [hunk 4]
```diff
@@ -1,121 +0,0 @@
...
-\FrontMatter                    % The front matter
-\ThesisTitlePage                % The title page
-\begin{ThesisDeclaration}       % The declaration
-  \DeclarationText
-  \AdvisorName
-\end{ThesisDeclaration}
-
-\begin{ThesisThanks}            % The acknowledgements (optional)
-  I would like to thank my supervisor RNDr. Adam Rambousek, Ph.D., Mgr. Jiri Kremser,
-  family and the people from Hawkular team. Next, my thanks goes to Ing. Daniel Němec, Ph.D.
```
Comment (grammar-review) [inline doc/thesis.tex:85]: Hawkular Team.

File: doc/content2.tex [hunk 5]
```diff
@@ -1,723 +0,0 @@
...
-                \scalebox{0.65}[0.5]{\includegraphics[angle=0]{img/src/results-overall-mse.pdf}}
-                \caption{Aggregated test results for all models. Average values of MSE for all tests samples.}
-                \label{img:results-overall-mse}
-            \end{center}
-        \end{figure}
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\chapter{Conclusion} \label{chap:conclusion}
-The goal of the master's thesis is to develop independent module in Java language for providing alert prediction
-capabilities for open source project Hawkular.
```
Comment (grammar-review) [inline doc/content2.tex:689]: tu este mozes zvazit vsunutie slova ....open project called Hawkular.

File: doc/thesis.tex [hunk 6]
```diff
@@ -1,121 +0,0 @@
...
-  for consulting theory behind time series analysis.
-
-  Last but not least, I would like to thank to company Red Hat that provided me 
-  with great opportunity to work on this project.
```
Comment (grammar-review) [inline doc/thesis.tex:89]: alebo Project ?

File: doc/content.tex [hunk 7]
```diff
@@ -1,677 +0,0 @@
...
-capabilities of targeted models and compares produced statistical quantities.
-
-The third chapter briefly describes existing monitoring and management solutions which are to some extend
-Hawkular competitors. Java libraries for time series forecasting are also mentioned there.
-
-The fourth chapter describes implementation part of the thesis. It starts with core artifacts for time series
-forecasting and continues to building web application and integration into Hawkular.
-
-The last chapter is dedicated to the evaluation of the forecasting accuracy of the implemented solution. As a default
-implementation were chosen models from \texttt{forecast} package from language R.
```
Comment (grammar-review) [inline doc/content.tex:45]: Models from \texttt{forecast} package from language R were chosen as a default implementation.

File: doc/content.tex [hunk 8]
```diff
@@ -1,677 +0,0 @@
...
-
-    %%%%%%%%
-    \section{Hawkular} \label{sec:hawkular}
-    The implementation part of the master's thesis is developed as a part of an open source project
-    Hawkular\footnote{Available at \url{http://www.hawkular.org}}. Therefore the application architecture and used
-    technologies had to fit into the overall project design.
```
Comment (grammar-review) [inline doc/content.tex:51]: ..., therefore, ... / ciarka a spojit vety  (hmm, to je divne, aj pri tomto komente mam pocit, ze uz som ho pisala. ale moznoze to bol len nejaky jemu podobny).

File: doc/content.tex [hunk 9]
```diff
@@ -1,677 +0,0 @@
...
-
-    Hawkular is middleware monitoring and management platform developed by company Red Hat and independent community
-    of contributors. It is a successor of very successful RHQ\footnote{Available at
-    \url{https://rhq-project.github.io/rhq/}.} project, also known as JBoss Operations Network (JON). By monitoring,
-    it is meant that there are agents for diverse applications which push data to the server. These agents can also
-    execute application specific actions.
-
-    The monolithic architecture of the RHQ project was due to its size hard to maintain and lacking robust REST API
-    lead to fresh development of new application. Hawkular, in contrast, consists of several loosely coupled or even
-    independent applications. These independent components are much easier to maintain and what is more important they
```
Comment (grammar-review) [inline doc/content.tex]: important, ...

File: doc/content.tex [hunk 10]
```diff
@@ -1,677 +0,0 @@
...
-    Recently a large number of successful applications using neural networks for time series modeling show that they
-    can produce valuable results \cite{ann-forecasting-state-art}. There are several non trivial issues with
-    determining the appropriate architecture of the network. This has to be taken into account because it can
-    dramatically effect learning performance and forecasting accuracy \cite{ann-model-selecting}.
-    Besides the problems with selecting right architecture learning process of artificial neural network is much more
-    computationally expensive than selecting appropriate ARIMA or exponential smoothing model \cite{ann-forecasting}.
-
-    Because Hawkular forecasting engine should be capable of predicting thousands of metrics at the same time, models
-    based on neural networks would have too high computational requirements. Therefore they are not suitable for our
-    environment.
```
Comment (grammar-review) [inline doc/content.tex:317]: ... therefore they are not suitable..

File: doc/content.tex [hunk 11]
```diff
@@ -1,677 +0,0 @@
...
-    There are multiple variants of ADF test. Some of them leave out parts of equation \ref{eq:adf-test}. The most
-    important ones and widely used are:
-
-    \begin{itemize}
-        \item \emph{nc}\,--\,for regression with no constant nor time trend ($\beta t$)
-        \item \emph{c}\,--\,for regression with an intercept but no time trend ($\beta t$)
-        \item \emph{ct}\,--\,for regression with an intercept and time trend
-    \end{itemize}
-
-    Each of them is good for testing particular type of stationarity. For example for testing if there is trend
```
Comment (grammar-review) [inline doc/content.tex:382]: ...for testing - zvaz vymenit za: ...when testing...

File: doc/content.tex [hunk 12]
```diff
@@ -1,677 +0,0 @@
...
-    present in the time series \emph{c} version is the best choice.
-
-    The implementation of this test fits multiple linear regression model from equation \ref{eq:adf-test} and
-    calculates ADF statistics listed in \ref{eq:adf-stat}. SE denotes standard error of estimated $\hat{\gamma}$.
-
-    %%%%%%%%
-    \section{Seasonality Detection} \label{sec:period-detection}
-    Forecasting engine in Hawkular system does not have any inside information about period of a time series being
-    modelled. Therefore automatic period identification has to be implemented. In practice it is a difficult task and
```
Comment (grammar-review) [inline doc/content.tex:391]: modelled, therefore... / spoj vety.

File: doc/content.tex [hunk 13]
```diff
@@ -1,677 +0,0 @@
...
-    result often differs from correct period, specially if there is significant noise present in the series
```
Comment (grammar-review) [inline doc/content.tex:392]: especially /more official

File: doc/content.tex [hunk 14]
```diff
@@ -1,677 +0,0 @@
...
-    \cite{period-meteo}.
-
-    There are several approaches how to implement automatic period identification. The most used ones are based on
-    autocorrelation function (ACF) or spectral density \cite{period-hydman}. This work applies ACF method.
-
-    In the following Chart \ref{img:period-acf} ACF function of sine function is shown. The period of this
-    function is seven. There are patterns repeated every seven observations and it is decreasing to zero.
```
Comment (grammar-review) [inline doc/content.tex:399]: uz predtym som toto zvazovala: decreasing by som nahradila "decreases" - kvoli tomu, ze je to proste fakt

File: doc/thesis.tex (hunk 6 above)
Comment (pavolloffay) [inline doc/thesis.tex:89]: To neviem,.. dava sa to takto s velkym? 

myslim ze aj s malym je to v poriadku

File: doc/content.tex (hunk 8 above)
Comment (pavolloffay) [inline doc/content.tex:51]: asi podobne, toho tam bola fura

File: doc/content.tex (hunk 10 above)
Comment (pavolloffay) [inline doc/content.tex:317]: Tazke: `requirements. Therefore they are not suitable for our  environment.`
Alebo si chcela povedat: `requirements, therefore, they are not suitable for our  environment.``` ?

File: doc/content.tex (hunk 12 above)
Comment (pavolloffay) [inline doc/content.tex:391]: takto: `modelled, therefore, automatic period identification`  ?

File: doc/thesis.tex (hunk 6 above)
Comment (grammar-review) [inline doc/thesis.tex:89]: no praveze sa dava. lebo je jeden, je jedinecny a konkretny. ale okej, moze byt aj malym.

File: doc/content.tex (hunk 14 above)
Comment (pavolloffay) [inline doc/content.tex:399]: takze takto: `seven observations and it decreases to zero.` ?

File: doc/content.tex (hunk 8 above)
Comment (grammar-review) [inline doc/content.tex:51]: ano, je to mozne :-)

File: doc/content.tex (hunk 10 above)
Comment (grammar-review) [inline doc/content.tex:317]: Tvoj druhy riadok :-)

File: doc/content.tex (hunk 12 above)
Comment (grammar-review) [inline doc/content.tex:391]: ano

File: doc/content.tex (hunk 14 above)
Comment (grammar-review) [inline doc/content.tex:399]: ano

Comment (grammar-review) [inline doc/content.tex:399]: ano

**Pull request closed without merging**